content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -178,36 +231,36 @@ The requirements to run TrueCommand in Docker Desktop for Windows are:
 * CPU support for VM Monitor Mode Extension (VT-c on Intel CPUs).
 * Hyper-V is enabled in Windows 10.
 * 4 GB memory at minimum.
-* [Docker Desktop](https://www.docker.com/products/docker-desktop) needs to be installed in Windows.
+* [Docker Desktop](https://www.docker.com/products/docker-desktop) must be installed in Windows.
 
 ### Enable Hyper-V
 
-To enable Hyper-V, click on Windows Start button and select or search for *Apps & Features*.
-Select *Programs and Features* under **Related settings** and *Turn Windows Features on or off* (Administrator action).
-Set the Hyper-V option and click *Ok*.
- You will need to restart your restart your computer for the change to take effect.
+Open the Start menu and search for Apps & Features.
+Select Programs and Features under Related settings and Turn Windows Features on or off (Administrator action).
+Set the Hyper-V option and click **Ok**.
+Restart your computer for the change to take effect.
 
 ![EnableHyperV](/images/TrueCommand/EnableHyperV.png "Enable HyperV")
 
-Alternatively, Hyper-V can be enabled with the **Powershell**.
-To do this, run Powershell as a Windows Administrator and enter `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`.
-If an error is returned that says the command could not be found, verify that you are running PowerShell as the Administrator.
+Alternatively, you can enable Hyper-V with Powershell.
+Run Powershell as a Windows Administrator and enter `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`.
+If Powershell returns an error saying it couldn't find the command, verify that you are running PowerShell as the Administrator.
```

Comment:
is it only one and only one administrator for Windows, or could there be several administrator accounts like we have in our software?
```suggestion
If Powershell returns an error saying it couldn't find the command, verify that you are running PowerShell as administrator.
```